packages/gamut/src/GridForm/__tests__/GridForm-test.tsx

```diff
@@ -19,44 +17,63 @@ import {
   stubTextField,
 } from './stubs';
 
+const fields = [stubCheckboxField, stubSelectField, stubTextField];
+
+const renderView = setupRtl(GridForm, {
+  fields,
+  showRequired: false,
+  submit: { type: 'fill', contents: <>Submit</>, size: 6 },
+});
+
+type renderViewReturn = ReturnType<typeof renderView>;
+
+const asyncRenderView = async (
+  props: Partial<
+    GridFormProps<Record<string, string | boolean | FileList | undefined>>
+  >
+): Promise<renderViewReturn> => {
+  let renderResults: renderViewReturn;
+
+  await act(async () => {
+    renderResults = await renderView(props);
```

Comment:
Nit: a little less hardcode-y

```suggestion
const asyncRenderView = async (
  ...props: Parameters<typeof renderView>
): Promise<renderViewReturn> => {
  let renderResults: renderViewReturn;

  await act(async () => {
    renderResults = await renderView(...props);
```